### DISCUSSION TITLE
v2-layout questions and comments - February 2024

### DISCUSSION THREAD

**Discussion by JC5** [firefly-iii/firefly-iii#8491] [General]
Hi there,

Please use this GitHub discussion to report all your Firefly III v2 layout issues and problems, and see if a fix is perhaps already available. Due to the volatile nature of software development, the v2 layout may be buggy or broken. As such the rule is: if it breaks you get to keep both parts. However, I am of course keen to fix these issues so please report them here.

This discussion will be closed and recreated every month, depending on the number of replies and issues etc.

To enable the v2 layout installation in your installation, please set `FIREFLY_III_LAYOUT=v2` in your environment variables. Be aware that pages may be broken, functionality is missing and data loss may occur when you use the layout to submit or change your data.

- [How to enable debug mode](https://docs.firefly-iii.org/how-to/general/debug/)

**Comment by JC5** [comment]
From the previous discussion:

- @not1q84-1: this has not yet been fixed, I've been fighting with the tag code for a while now. Stay tuned.
- @absentbri: no PWA manifest yet, it's on the list but not very high.
- @pelaxa: updating transactions should work.
- @SplinterHead: your issue about the empty box is not yet fixed

**Comment by tmeekes** [comment]
Just found that you're working on a new layout. Just enabled it. I like the look of most of the improvements (not convinced about the shift of the search bar just yet, but I'll probably warm up to it).

One thing I'd really like to see, that I find difficult to extract quickly right now is being able to analyze income vs budgeted / non-budgeted expenses., either on the dashboard or through reports. I can kind of extrapolate it after some work, but there isn't anything that enables it in a few clicks.

The reason I mention it, is that there's pretty good mechanisms for tracking the budgeted transactions... but not everything is always budgeted. Sometimes you have one-off expenses that you might cover from savings or somehow else, which was not a budgeted event. Being able to showcase those I think is important for capturing alongside the budgeted activity, especially to provide an accurate ratio of income/expense.

Just a consideration, appreciate everything you do.
